common/network/network.go

```diff
@@ -0,0 +1,77 @@
+package network
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/scroll-tech/go-ethereum/params"
+)
+
+// Network represents a known network
+type Network string
+
```

Comment:
As https://scrollco.slack.com/archives/C02PKAQ2430/p1709044228395759 discussed, maybe we reach a consensus：don't import the genesis.json for `rollup_relayer` and `coordinator`. 

And if really need this, coordinator will also use this. BTW, I think just add a config option to currently config.json will also  achieve the same effect. How do you think? @colinlyguo @Thegaram

Reply:
> maybe we reach a consensus：don't import the genesis.json for rollup_relayer and coordinator.

I believe, in that thread, the consensus was to use the genesis config, which is what this PR does.

Reply:
oh okay, maybe i don't express the opinion clearly in the thread. actually my mean that we don't need import another config file. And we just use one option from the genesis.json, besides I think the current option (like config.HomesteadBlock, config.DAOForkBlock and etc) can't satisfied, maybe add other option like `config.ScrollAprilBlock`. if so, why not import the option to config.json

Reply:
Synced with @omerfirmak , I think the way is make sense.